open_humans/urls.py

```diff
@@ -137,6 +141,11 @@
     path('account/login/oauth2/', views.OAuth2LoginView.as_view(),
          name='account-login-oauth2'),
 
+    # use our own template.
+    path('account/social/signup/',
+         account_views.SocialSignupView.as_view(),
+         name='SocialSignupView'),
+
     path('account/', include('allauth.urls')),
```

Comment:
Examining this... it looks like it wasn't implemented correctly as a fallback here in the first place, these are wonky paths here.

![image](https://user-images.githubusercontent.com/82631/49249459-0b69d600-f3d1-11e8-8a95-46b6bea314fc.png)